internal/backend/crypto/gpg/cli/keyring.go

```diff
@@ -55,16 +59,26 @@ func (g *GPG) Fingerprint(ctx context.Context, id string) string {
 // - EmailFromKey: {{ .Email }}.
 func (g *GPG) FormatKey(ctx context.Context, id, tpl string) string {
 	if tpl == "" {
-		return g.findKey(ctx, id).OneLine()
+		k, found := g.findKey(ctx, id)
+		if !found {
+			return ""
+		}
+		return k.OneLine()
 	}
 
 	tmpl, err := template.New(tpl).Parse(tpl)
 	if err != nil {
 		return ""
 	}
 
+	var gid gpg.Identity
+	k, found := g.findKey(ctx, id)
+	if found {
```

Comment:
Else what?

Reply:
gid will be left at the zero value of `gpg.Identity`.